pkg/openshift/certgen/unstable/templates/master/tmp/ansible/azure-local-master-inventory.yml

```diff
@@ -15,11 +15,11 @@ localmaster:
 
     # NOTE: This changes in 3.10+ and it's all combined into a single variable
     #       under the name openshift_web_console_image_name
-    openshift_web_console_prefix: "IMAGE_PREFIX/"
-    openshift_web_console_image_name: "IMAGE_TYPE-web-console"
-    openshift_web_console_version: "vVERSION"
+    openshift_web_console_image_name: "IMAGE_PREFIX/IMAGE_TYPE-web-console:vVERSION"
```

Comment:
delete comment above

Reply:
done.

Reply:
comment still needs to get deleted

Reply:
@kwoodson delete `# NOTE: This changes in 3.10+ and it's all combined into a single variable under the name openshift_web_console_image_name`

Reply:
done.